en/core-libraries/httpclient.rst

```diff
@@ -469,13 +469,14 @@ instead. You can force select a transport adapter using a constructor option::
 Testing
 =======
 
-.. php:trait:: Cake\TestSuite\HttpClientTrait
+.. php:trait:: use Cake\Http\TestSuite\HttpClientTrait
```

Comment:
That `use` shouldn't be there, this is Sphinx stuff, not PHP. It might have been wrong before too, I'm not sure if a `php:namespace::` is always needed (@markstory ?), but the qualified name wasn't picked up correctly.

Reply:
Yes sorry, it's a bad copy/paste.
Just Http was missing in the path

Reply:
I didn't look closely since it was similar to the other PR.

Reply:
Oh, I didn't even notice that there were two PRs, guess the identical titles fooled me.